src/server/status_server/mod.rs

```diff
@@ -378,8 +376,11 @@ where
             }
         };
 
+        let svg_content_type: hyper::http::HeaderValue =
+            hyper::http::HeaderValue::from_str("image/svg+xml").unwrap();
```

Comment:
How about adding a query like "output=svg"? Sounds like this is more user friendly (which can be invoked in the browser).

Reply:
Anyway this is a product change. I would like to receive more comments about this kind of change.